plasmic.json

```diff
@@ -106,5 +106,5 @@
   "globalVariants": {
     "variantGroups": []
   },
-  "cliVersion": "0.1.65"
+  "cliVersion": "0.1.67"
 }
```

Comment:
Nitpick, can we have an "end of line" character? Not sure if this comes messed up from Plasmic and if we can even change this file ourselves.